src/Status/StatusCard/StatusCard.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+
+import { Box, Card, CardContent, Typography } from "@mui/material";
+
+import { StatusCardProps } from "./StatusCard.types";
+import StatusIcon from "../StatusIcon/StatusIcon";
+import { grey } from "@mui/material/colors";
+
+// custom card component that will be used to display status information
+const card = ({ status, name, width, height }: StatusCardProps) => {
```

Comment:
Again, whats the use-case for being able to set height/width? I can't see a need to do it in Figma. Also confusing that its icon height/width and not card.

Reply:
Removed height/width instead added height/width to the icon as per figma